emily/api-definition/package.json

```diff
@@ -3,7 +3,7 @@
   "scripts": {
     "build-openapi": "smithy build",
     "clean-openapi": "smithy clean",
-    "build-rs": "openapi-generator-cli generate -i ../../.generated-sources/smithy/openapi/openapi/Emily.openapi.json -g rust -c rust-config.json -o ../../.generated-sources/emily",
+    "build-rs": "npx @openapitools/openapi-generator-cli generate -i ../../.generated-sources/smithy/openapi/openapi/Emily.openapi.json -g rust -c rust-config.json -o ../../.generated-sources/emily",
```

Comment:
This is necessary to prevent people from needing to install the `openapi-generator-cli` locally.

Reply:
@hstove FYI, feel a bit bad about reversing this change especially since I approved it.